### PR TITLE
Add Tamil translation

### PR DESCRIPTION
**Pull request by Jaiganeshkumaran**
Adds translation for Tamil (language code TA)

### CODE REVIEW

**Comment by dpaulino** [comment]
There's been a few new strings. Do you mind updating?

**Comment by dpaulino** [comment]
I think there's an issue with one of the strings

**Comment by Jaiganeshkumaran** [comment]
@dpaulino I've fixed that issue. Can you merge this now?

**Comment by dpaulino** [comment]
I think you're missing some new strings from the latest main branch. Can you pull from main?

**Comment by Jaiganeshkumaran** [comment]
> I think you're missing some new strings from the latest main branch. Can you pull from main?
I have translated them now as well.

File: AmbientSounds.sln [hunk 1]
```diff
@@ -9,7 +9,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AmbientSounds", "src\Ambien
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AmbientSounds.Web", "src\AmbientSounds.Web\AmbientSounds.Web.csproj", "{7816F10F-CF63-4E0E-8186-D51E146EDE2F}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AmbientSounds.Tests", "AmbientSounds.Tests\AmbientSounds.Tests.csproj", "{190FE594-FDCE-4707-B01C-EE7C39C541F3}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AmbientSounds.Tests", "AmbientSounds.Tests\AmbientSounds.Tests.csproj", "{190FE594-FDCE-4707-B01C-EE7C39C541F3}"
```
Comment (dpaulino) [inline AmbientSounds.sln:12]: Can you revert this?

File: AmbientSounds.sln [hunk 2]
```diff
@@ -26,6 +26,8 @@ Global
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{B0BA021E-FA43-4921-A25A-0004A43F7C40}.Debug|Any CPU.ActiveCfg = Debug|x86
+		{B0BA021E-FA43-4921-A25A-0004A43F7C40}.Debug|Any CPU.Build.0 = Debug|x86
+		{B0BA021E-FA43-4921-A25A-0004A43F7C40}.Debug|Any CPU.Deploy.0 = Debug|x86
```
Comment (dpaulino) [inline AmbientSounds.sln:30]: Can you revert this?

File: src/AmbientSounds.Uwp/AmbientSounds.Uwp.csproj [hunk 3]
```diff
@@ -11,7 +11,7 @@
     <AssemblyName>AmbientSounds.Uwp</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.18362.0</TargetPlatformVersion>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.19041.0</TargetPlatformVersion>
```
Comment (dpaulino) [inline src/AmbientSounds.Uwp/AmbientSounds.Uwp.csproj:14]: Revert as well please?

**Review by dpaulino — CHANGES_REQUESTED** [review]

**Comment by dpaulino** [comment]
@Jaiganeshkumaran Very sorry for the very long review times. This is my one last requested change. Once this is done, we'll merge ASAP. And if you can get the changes done within the next 16 hours, we can get the translation included in the update coming on Monday

**Review by dpaulino — APPROVED** [review]
Thanks!

**Pull request merged**